lib/Panda.pm

```diff
@@ -237,6 +237,27 @@ class Panda {
             when 'look'    { self.look($bone) };
         }
     }
+
+    method version {
+        my $version;
+
+        if '.git'.IO.e {
```

Comment:
Wouldn't this give the version information of any repository that's currenly your $*CWD?